tests/chocolatey-tests/commands/choco-upgrade.Tests.ps1

```diff
@@ -272,7 +272,10 @@ To upgrade a local, or remote file, you may use:
             $Output = Invoke-Choco upgrade upgradepackage --confirm
         }
 
-        It 'Exits with Success (0)' {
+
+        # `upgradepackage` contains a beforeModify that throws, which triggers an incorrect -1 exit code.
+        # See https://app.clickup.com/t/20540031/PROJ-615
+        It 'Exits with Success (0)' -Pending {

```

Comment:
Don't mark it as Pending, but rather with
```suggestion
        It 'Exits with Success (0)' -Tag Broken {

```

Reply:
Thanks! Fixed :)